DevoxxClientMobile/src/main/java/com/devoxx/views/helper/Util.java

```diff
@@ -114,6 +115,28 @@ public static FloatingActionButton createWebLaunchFAB(Supplier<String> urlSuppli
             });
         });
     }
+    
+    public static void requestRating() {
+        String url = null;
+        if(Platform.isAndroid()) {
+            String packageName = "com.devoxx";
+            url = "market://details?id=" + packageName;
+        } else if (Platform.isIOS()) {
+            String appId = "1094805620";
+            url = "itms-apps://itunes.apple.com/WebObjects/MZStore.woa/wa/viewContentsUserReviews?id=" + appId +"&amp;onlyLatestVersion=true&amp;pageNumber=0&amp;sortOrdering=1&amp;type=Purple+Software";
```

Comment:
This doesn't work with iOS 11. Check this: 
https://stackoverflow.com/a/46335171/3956070